packages/web-wallet/playwright.config.ts

```diff
@@ -0,0 +1,42 @@
+import { defineConfig, devices } from '@playwright/test';
+
+export default defineConfig({
+  testDir: './tests',
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: 'html',
+  use: {
+    baseURL: 'http://localhost:5173',
+    trace: 'on-first-retry',
+  },
+
+  projects: [
+    {
+      name: 'chromium',
+      use: {
+        ...devices['Desktop Chrome'],
+        // Add extension support when available
+        // args: ['--load-extension=./extensions/metamask-flask'],
+        // headless: false,
```

Comment:
Should we add these dependencies or not?

Reply:
Removed those... This was from an early attempt to make playwright tests

But the current PR is big enough as is, we can go back to this in a later PR

Reply:
Issue created: https://github.com/HathorNetwork/hathor-rpc-lib/issues/108